lib/Echidna/UI.hs

```diff
@@ -243,12 +244,12 @@ stopWorkers :: MonadIO m => [(ThreadId, a)] -> m ()
 stopWorkers workers =
   forM_ workers $ \(threadId, _) -> liftIO $ killThread threadId
 
-vtyConfig :: IO Config
+vtyConfig :: IO VtyUserConfig
 vtyConfig = do
-  config <- Vty.standardIOConfig
-  pure config { inputMap = (Nothing, "\ESC[6;2~", EvKey KPageDown [MShift]) :
-                           (Nothing, "\ESC[5;2~", EvKey KPageUp [MShift]) :
-                           inputMap defaultConfig }
+  config <- Vty.userConfig
```

Comment:
Note:

The standardIOConfig has some unix specific configurations. From 6.x, it is gone. The closest is a function from vty-unix package.